### DISCUSSION TITLE
Offline Reader

### DISCUSSION THREAD

**Discussion by omodia** [mienaiyami/yomikiru#168] [General]
Do you plan to add  [Anilist Tracking](https://github.com/mienaiyami/yomikiru/issues/167)?

If you plan to add that feature,
- A humble request from me,
- Please close the before issues and please release a version without that feature, before adding that anilist tracking feature.

**Comment by zeoint** [comment]
I feel the same developer. I feel you omodia.

**Comment by mienaiyami** [comment]
I might just block internet for whole app (except for the update) by default. 
But still, it is not the final solution yet.

**Reply by zeoint** [reply]
Cool.

**Reply by omodia** [reply]
Okay.

**Comment by mienaiyami** [comment]
https://github.com/mienaiyami/yomikiru/issues/190#issuecomment-1609124827
